src/main/java/uk/gov/justice/digital/hmpps/keyworker/controllers/KeyworkerStatsController.java

```diff
@@ -59,4 +61,34 @@ public KeyworkerStatsDto getStatsForStaff(
 
     }
 
+    /* --------------------------------------------------------------------------------*/
+
+    @ApiOperation(
+            value = "Get Keyworker stats at specified prison.",
+            notes = "Get Keyworker stats at specified prison.",
+            nickname="getKeyworkerStats")
+
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "OK", response = PrisonStatsDto.class),
+            @ApiResponse(code = 400, message = "Invalid request.", response = ErrorResponse.class ),
+            @ApiResponse(code = 404, message = "Requested resource not found.", response = ErrorResponse.class),
+            @ApiResponse(code = 500, message = "Unrecoverable error occurred whilst processing request.", response = ErrorResponse.class) })
+
+    @GetMapping(path = "/{prisonId}")
+    public PrisonStatsDto getKeyworkerStats(
```

Comment:
I wonder if a better name would be getStatsForPrison, or getPrisonStats?

Reply:
Agreed with change